portal/views/commits.py

```diff
@@ -0,0 +1,25 @@
+
+
+from django.http import HttpRequest, HttpResponse
+from django.template.response import TemplateResponse
+from django.shortcuts import get_object_or_404
+from django.contrib.auth.decorators import login_required
+from portal.services import github
+from django.core.cache import cache;
+from portal.tasks import get_commits;
+
+@login_required
+def commits_index(request: HttpRequest) -> HttpResponse:
+    repos = request.user.get_active_enrollment().project.repositories.all()
```

Comment:
Let's handle the case where `request.user.get_active_enrollment() is None` (the logged in user is not enrolled at the moment because its the start of a semester or between semesters)